war/pom.xml

```diff
@@ -95,12 +95,6 @@
             <classifier>tests</classifier>
             <scope>test</scope>
         </dependency>
-        <!-- required by GDocs-->
-        <dependency>
-            <groupId>commons-lang</groupId>
-            <artifactId>commons-lang</artifactId>
-            <version>2.6</version>
-        </dependency>
```

Comment:
This was automatically brought into the PR when creating. This change was added to master in (# #528)